### ISSUE TITLE
Add support for loans (Schedule D)

### ISSUE THREAD

**Issue by dwillis** [dwillis/itemizer#10]

**Comment by bycoffe** [comment]
I created a table layout for Schedule D items: https://github.com/bycoffe/itemizer/commit/6753a05fa837e16f69fc2e0464c9cb75be1a6cc2

Any thoughts about the best way to link to the Schedule D page from the homepage?

**Comment by dwillis** [comment]
One way might be to replace the "Receipts" and "Spending" cols with a single column that has Schedule links, like so:

A | B | C | D

(with or without the pipes). The list would depend on the form type.

**Comment by bycoffe** [comment]
I find it useful to be able to sort by "Receipts" and "Spending". I do like the idea of listing the schedules, though.

**Comment by dwillis** [comment]
Yeah, that's true - we should keep Receipts and Spending, and maybe reduce the amendment column to make room for list of schedules?

**Comment by bycoffe** [comment]
We could also narrow the Committee column. Many of them go to two lines anyway, and they're still readable if we limit it to about 300px.

Is there a mapping of form types to schedules? I'd originally thought we'd just show the form types included in the specific filing, but that would mean parsing every filing on a given day.

**Comment by dwillis** [comment]
I don't think there is, but shouldn't be too hard to do. Probably should be in Fech as well.
